.gitignore

```
@@ -29,6 +29,7 @@
 /light-sdk-ts/node_modules/
 /light-sdk-ts/lib/
 /light-sdk-ts/lookUpTable.txt
+/light-sdk-rs/docs
```

```suggestion
/light-sdk-ts/docs
```

You have a typo here. After fixing it, make sure you remove the docs from your commit by doing

```
git rm -rf light-sdk-ts/docs
```

(and then include it in your commit with `git commit --amend`)

---

yea, i fixed it in my local branch